src/main/java/ua/danit/final_project/repositories/EmployeeRepo.java

```diff
@@ -0,0 +1,10 @@
+package ua.danit.final_project.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.data.jpa.repository.Query'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=39&issues=AWUu39d4W2AOP0Ck9j6i&open=AWUu39d4W2AOP0Ck9j6i)

[ISSUE_KEY]: AWUu39d4W2AOP0Ck9j6i